Units/parser-c.r/bug1458930.c.d/expected.tags

```diff
@@ -1,2 +1,2 @@
-x	input.c	/^char x();$/;"	p	file:
-y	input.c	/^wchar_t y();$/;"	p	file:
+x	input.c	/^char x();$/;"	p
```

Comment:
As I wrote basically you can decide it; make ctags suitable to your tool.
When you are not sure what to do, keep the compatibility.